source/scpd/types/Utils.d

```diff
@@ -18,6 +18,10 @@ import scpd.types.Stellar_SCP;
 import scpd.types.Stellar_types;
 import scpd.types.XDRBase;
 
+// todo fix: SCPQuorumSet should be const, but triggers a wild vibe.d error
```

Comment:
What's the error again ? Is it RT or CT ?

Reply:
The issue is I can't reproduce it anymore. /shrug